ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/datepicker/v1/datepicker/_cq_dialog/.content.xml

```diff
@@ -104,18 +105,30 @@
                             jcr:title="Formats"
                             sling:resourceType="granite/ui/components/coral/foundation/container">
                         <items jcr:primaryType="nt:unstructured">
+                            <displayPatternGroup
+                                    jcr:primaryType="nt:unstructured">
+                                    <items jcr:primaryType="nt:unstructured">
+                                        <displayPictureClause
+                                                jcr:primaryType="nt:unstructured"
+                                                sling:resourceType="granite/ui/components/coral/foundation/form/textfield"
+                                                fieldLabel="Display format"
+                                                fieldDescription="The format in which the date is displayed"
```

Comment:
We should mention it is a regular expression based on ..

Reply:
This is not a regular expression, but we should add documentation of our formatting symbols.